### DISCUSSION TITLE
Nightscout entries download failed: timeout

### DISCUSSION THREAD

**Discussion by D-S-Mason** [NightscoutFoundation/xDrip#3006] [Q&A]
I am using a Libre2 (Australia) -> Diabox -> Xdrip+ setup

I am using xdrip in nightscout follow mode with http://localhost :17580, and had been doing so for many weeks on my Samsung Galaxy S22, unlt one day it started intermittently giving me the error:

"NS entries download Failed:
java.net.SocketTimeoutException: timeout"

Then after a few days of that error breing intermittent, it became permanent.

This was back in April, and at that time, I started using the same setup on another phone I had available to me, a Samsung Galaxy S10 5G.  This was working great until a few nights ago.  After the sensor had expired for 12 hours or so, I replaced it, but was hit with the same error message on the other phone.

Can anyone please tell me what is causing thte issue and is there any way I can resolve it?

**Comment by Navid200** [comment]
If xDrip has readings and cannot upload to Nightscout, it really doesn't matter where the readings come from.  So, never mind Libre 2.

Are you sure your Nightscout server is healthy?

**Reply by D-S-Mason** [reply]
I am trying to download Nightscout not upload, and it is localhost from Daibox.  I can browse to the localhost address with my phone browser, and it all seems in order as far as I can tell.

**Reply by Navid200** [reply]
Please post a screenshot of the hardware data source page.
https://navid200.github.io/xDrip/docs/HardwareDataSource.html

**Reply by D-S-Mason** [reply]
Hardware Data Source, as well as other screenshots
![Hardware source](https://github.com/NightscoutFoundation/xDrip/assets/142202785/e361ef18-af4d-4a9b-8b59-7b3f4d82d61d)
![NS Follow URL](https://github.com/NightscoutFoundation/xDrip/assets/142202785/9e50dc03-9324-4bcc-a932-7107cecb4f09)
![NS Entries at localhost 17580](https://github.com/NightscoutFoundation/xDrip/assets/142202785/453281a1-b58f-4507-acd2-99a8e922e3dc)
![System Status](https://github.com/NightscoutFoundation/xDrip/assets/142202785/a029d151-ae33-4305-a961-727805e2fdd5)
![System Status 2](https://github.com/NightscoutFoundation/xDrip/assets/142202785/d95190e7-dcc8-4fc3-9ca8-b1c53601a64a)

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
By any chance are you also using the xDrip+ web server? https://xdrip.readthedocs.io/en/latest/use/interapp/#web-service

**Reply by D-S-Mason** [reply]
No, the xDrip+ web server is off.  (also turning it on didn't help either)
